src/main/java/FileContent.java

```diff
@@ -0,0 +1,20 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileContent implements Content {
+    private final File file;
+    public FileContent(final File f) {
+        this.file = f;
+    }
+
+    public String read() throws IOException {
+        return new String(Files.readAllBytes(Paths.get(file.getAbsolutePath())), "UTF-8");
+    }
+
+
+    public void save(String content) throws IOException {
```

Comment:
@ArthurRomasyuk why not stick to the terminology `read/write`? In other words `save` vs `write`?